src/key-value-engine/B-Tree-vs-Log-Structured-Merge-Tree.md

```diff
@@ -0,0 +1,130 @@
+# B-Tree vs Log-Structured Merge-Tree
+
+[B-tree](https://en.wikipedia.org/wiki/B-tree) and [Log-Structured Merge-tree](https://en.wikipedia.org/wiki/Log-structured_merge-tree) (LSM-tree) are the two most widely used data structures for data-intensive applications to organize and store data. However, each of them has its own advantages and disadvantages. This article aims to use the quantitative approaches to compare these two data structures. 
+
+## Metrics
+
+In general, there are three most important metrics to measure the performance of a data structure, which include write amplification, read amplification, and space amplification. This section aims to describe these metrics. 
```

Comment:
`most` doesn't really fit in here. Maybe `there are three most important metrics` ... `there are three critical metrics`, or...

* crucial
* important (without most)
* valuable